doc/man/local_lsmplugin.1.in

```diff
@@ -0,0 +1,89 @@
+.TH local_lsmplugin "1" "July 2017" "local_lsmplugin @VERSION@" "libStorageMgmt"
+.SH NAME
+local_lsmplugin -- LibstorageMgmt Local Pseudo plugin
+
+.SH DESCRIPTION
+LibstorageMgmt local plugin allows user to manage locally attached storage
+hardware/software without caring which real plugin(s) should be used.
+The 'local_lsmplugin' executable file is for libStorageMgmt
+daemon to execute when client user specifies local plugin in the URI.
+
+Currently, local plugin is wrapping of these plugins:
+
+    * MegaRAID plugin -- `megariad_lsmplugin(1)`.
+    * SmartArray plugin -- `hpsa_lsmplugin(1)`.
+    * Adaptec plugin -- `arcconf_lsmplugin(1)`.
+
+Certain binrary tool might be required, please check error message or manpage
+of above plugins.
+
+Extra SELinux actions are required to allowing plugin access the
+hardware/software.
+
```

Comment:
I believe I should remove that line. LSM works well on Fedora and RHEL. That was an old line when SELinux block external binrary tools of storcli and hpssacli.